index.bs

```diff
@@ -3649,7 +3673,10 @@ The serialization algorithm works by appending successive byte strings to an, in
     1. Append 0x66616c7365 (`false`) to |result|.
```

Comment:
In the introduction of this section (emphasis added):

>This verification algorithm requires only [...] and **three** conditional checks [...]

The "three" needs to either be deleted or updated to four. Something like "simple conditional checks" could work too.

Reply:
Fixed, thanks!